src/serac/infrastructure/logger.cpp

```diff
@@ -9,12 +9,17 @@
 #include "serac/infrastructure/initialize.hpp"
 #include "serac/infrastructure/terminator.hpp"
 
+#include <fstream>
+
 namespace serac::logger {
 
 static int logger_rank = 0;
 
 int rank() { return logger_rank; }
 
+// output stream for the SLIC LogStreams that write to a file
+static std::unique_ptr<std::ostream> logger_ostream;
```

Comment:
Can this be just a `std::ofstream`?

Reply:
I swear i tried this originally.. but whatever i did before clearly was wrong but this works now.

Reply:
I swear i tried this originally but it failed for some reason.  But going back and trying again it worked.  I must have messed something else up.

Thanks.